### DISCUSSION TITLE
[Suggestion] Favorites or ratings

### DISCUSSION THREAD

**Discussion by mijolen** [Difegue/LANraragi#382] [Ideas]
Currently LRR doesn't store any user info on server side, so when it will be implemented, It would be nice to have in GUI simple method to rate items. Yes, it could be done through tags, but only admin can modify them which is a bit bothersome to do. Besides it applies to all users.

**Comment by Difegue** [comment]
Hey there 👋

I currently don't plan to add specific support for ratings so there's no changes on that front, but if you do go for the tags route with something like `rating:⭐⭐⭐⭐`, you'll be able to customize the display and sort by rating in a future release thanks to https://github.com/Difegue/LANraragi/issues/195.  

I still recommend using a category for favorites.  
It's true that both of those options are only editable when logged-in, but the user experience is very much tailored towards single-user scenarios at the moment anyways -- Multi-user has never been asked for much and would require quite a bit more work. (https://github.com/Difegue/LANraragi/discussions/111)

**Reply by mijolen** [reply]
Hello.
Thank you for the answer. It seems the best option would be to try implement multi-user support by myself.
